source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -2199,8 +2199,8 @@ bool COpenGLDriver::SAuxContext::setActiveTexture(uint32_t stage, core::smart_re
 
     if (CurrentTexture[stage])
     {
-        if (CurrentTexture[stage]->getVirtualTextureType()!=IVirtualTexture::EVTT_BUFFER_OBJECT&&
-            CurrentTexture[stage]->getVirtualTextureType()!=IVirtualTexture::EVTT_2D_MULTISAMPLE)
+        if (CurrentTexture[stage]->getVirtualTextureType()!=IRenderableVirtualTexture::EVTT_BUFFER_OBJECT&&
```

Comment:
EVTT_BUFFER_OBJECT should disappear from the engine

Reply:
@Crisspl ?

Reply:
in upcoming commit